webrtc.html

```diff
@@ -5152,6 +5224,45 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       on an <code>RTCRtpReceiver</code> are modified, a negotiation is
       triggered to signal the changes regarding what the application wants to
       receive to the other side.</p>
+      <p>To <dfn>create an RTCRtpReceiver</dfn> with kind, <var>kind</var>, and
+      optionally an id string, <var>id</var>, run the following steps:</p>
+      <ol>
+        <li>
+          <p>Let <var>sender</var> be a new <code><a>RTCRtpSender</a></code>
+          object.</p>
+        </li>
+        <li>
+          <p>Let <var>track</var> be a new <code><a>MediaStreamTrack</a></code>
+          object [[!GETUSERMEDIA]].</p>
+        </li>
+        <li>
+          <p>Initialize <var>track.kind</var> to <var>kind</var>.
+        </li>
+        <li>
+          <p>If an id, <var>id</var>, was given as input to this algorithm, let
+          <var>track.id</var> be <var>id</var>. (Otherwise the value generated
+          when <var>track</var> was created will be used.)</p>
```

Comment:
I guess *set* instead of *let* here? I guess not *initialize*, since we're referencing a generated value as the already-initialized value.

I'm a bit new to how we spec creation of public-facing objects without a public constructor, so I may be wrong here. Is there an established guideline for this?